src/app/usersearch/usersearch.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-usersearch',
+  templateUrl: './usersearch.component.html'
+})
+export class UsersearchComponent {
```

Comment:
why UsersearchComponent ? it should be called PatientSeachComponent?

Reply:
done